lib/ret_web/channels/hub_channel.ex

```diff
@@ -603,7 +618,7 @@ defmodule RetWeb.HubChannel do
         response
         |> Map.put(:session_id, socket.assigns.session_id)
         |> Map.put(:session_token, socket.assigns.session_id |> Ret.SessionToken.token_for_session())
-        |> Map.put(:subscriptions, %{web_push: is_push_subscribed})
+        |> Map.put(:subscriptions, %{web_push: is_push_subscribed, favorites: is_favorited})
```

Comment:
yeah the name is basically broken now -- i think the data structure is sane, but instead of subscriptions it should be some word that describes" the set of opt-in relationships this user has with this room